content/docs/enterprise/external-data/geoip.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: GeoIP
+description: Configure the GeoIP external data provider to extend your access policies.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The GeoIP integration provides provides a list of IP address ranges and their country of origin. This data provider could allow an administrator to restrict, or allow access based on a users' known GeoIP fingerprint.
+
+```json tittle="Example"
+[
+  {
+    "$index": { "cidr": "1.0.0.0/24" },
+    "id": "1.0.0.0/24",
+    "country": "US",
+    "state": "",
+    "city": "",
+    "zip": "",
+    "timezone": ""
+  },
+  {
+    "$index": { "cidr": "1.0.1.0/24" },
+    "id": "1.0.1.0/24",
+    "country": "CN",
+    "state": "",
+    "city": "",
+    "zip": "",
+    "timezone": ""
+  }
+]
+```
+
+:::caution
+The external data sources we provide are meant to be examples and inspiration for users to create their own data sources. We won't maintain these integrations in perpituity, and changes to the APIs they interact with may break them in the future.
+
+The [datasource](https://github.com/pomerium/datasource) project is open-source, and if the community wishes to provide contributions to keep it working in the future, we will shepherd those updates.
+:::
+
+## Install
+
+<Tabs>
+<TabItem value="compose" label="Docker Compose">
+
+These instructions assume a local testing environment using [Docker Compose]. Adjust as needed for your deployment environment.
+
+1. Download the IP2Location CSV GeoIP database from [ip2location.com].
+
+1. Add the datasource docker image to Docker Compose:
+
+```yaml title="docker-compose.yaml"
+services:
+  ip2location:
+    image: docker.cloudsmith.io/pomerium/datasource/datasource:main
+    container_name: ip2location
+    command:
+      - ip2location
+      - /var/ip2location.csv.zip
+    ports:
+      - 8080:8080
+    volumes:
+      - ${PWD}/IP2LOCATION-LITE-DB1.CSV.ZIP:/var/ip2location.csv.zip
+```
+
+1. Bring up the new container.
```

Comment:
In the preview, this entry is listed as "1." again instead of "3."

Not sure what the fix is besides manually listing it as 3. Pointing it out.

Reply:
Yeah, I've compared this with similar markdown in another file and I'm stupendously confused as to why the numbered list is not happening here. Leaving this open.

Reply:
This is a "feature" of markdown. It will auto-count for you.